### ISSUE TITLE
Question: Missing documentation / source code

### ISSUE THREAD

**Issue by MovGP0** [danielmiessler/Substrate#20]
As far as I understand, the `substrate` takes an argument and does something like this:

```mermaid
graph TD
    A((Start)) --> B[Split Argument into Claims]
    B --> C[Search for Sources]
    C --> D[Analyze Sources]
    D --> E[Summarize Findings]
    E --> F((End))
    D -->|Parallel Execution| G[Individual Source Analysis]
    G --> D
```

But so far I was unable to find the relevant source code or documentation on this. Where can I find the source/documentation?

**Comment by ktneely** [comment]
I agree, and I think it would be super helpful to have a kind of sandbox to test how the system works before creating PRs for the primary project.  I can also think of some company-internal or private use cases for this framework. 

I know the project just consolidated many repos into one, but I think it would be helpful to have a "substrate-template" repo that is essentially the bare framework + a getting started how-to.   That would be super helpful.

**Comment by danielmiessler** [comment]
I think the repo itself could be a template, AI should be able to do that for you once things get built out a bit more. But yes, perhaps we could make a Naked template system as well.
